templates/core/terraform/scripts/build-deploy-ui.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
```

Comment:
Can you please add all 3 options:
```bash
set -o errexit
set -o pipefail
set -o nounset
```

Not having them creates real issues when we chain scripts.